deployments/nginx-behind-envoy-deployment.yaml

```diff
@@ -106,22 +116,18 @@ spec:
             capabilities:
               add: ["IPC_LOCK"]
           command:
-            - "/envoy-static"
+            - "/usr/local/bin/envoy-static"
           args:
             - "-c"
             - "/etc/envoy/config/envoy-conf.yaml"
             - "--cpuset-threads"
```

Comment:
`entrypoint.sh`  gets used if we drop both `command` and `args`. That might be better approach now that we have only one image.

Reply:
Agree

Reply:
Agree